plugins/content/caption/base/lib/api/KalturaCaptionAsset.php

```diff
@@ -61,6 +61,24 @@ class KalturaCaptionAsset extends KalturaAsset
 	 */
 	public $status;
 	
+	/**
+	 * The parent id of the asset
+	 *
+	 *@var string
+	 *@readonly
+	 *
+	 */
+	public $parentId;
+	
+	/**
+	 * Is multi-language caption asset
+	 *
+	 *@var bool
+	 *@readonly
+	 *
+	 */
+	public $isMulti;	
```

Comment:
1. Why is it readonly?
2. Why won't you enable a new language enum value that called multilingual, or leave it null to mark it as multilingual.